src/main/java/agents/leeroy/GameUtils.java

```diff
@@ -118,13 +125,48 @@ private static Integer evaluateInitialBoard(Node node, Risk risk) {
         return continents;
     }
 
+    private static int getTotalContinentBonus(int playerId, RiskBoard riskBoard) {
```

Comment:
interesting method :-)

Reply:
Yeah, I wanted to have a uniform interface for bonus and malus, so I created this intermediate method :sweat_smile: